lib/galaxy/config/__init__.py

```diff
@@ -757,6 +757,9 @@ def parse_config_file_options(self, kwargs):
             user_preferences_extra_conf_path=[self._in_config_dir('user_preferences_extra_conf.yml')],
             workflow_resource_params_file=[self._in_config_dir('workflow_resource_params_conf.xml')],
             workflow_schedulers_config_file=[self._in_config_dir('workflow_schedulers_conf.xml')],
+            markdown_export_css=[self._in_config_dir('markdown_export.css')],
+            markdown_export_css_pages=[self._in_config_dir('markdown_export_pages.css')],
+            markdown_export_css_invocation_reports=[self._in_config_dir('markdown_export_invocation_reports.css')],
```

Comment:
@jmchilton Did you mean to ship those css files with Galaxy / forgot to git add ? Or is that meant as an optional thing ?
I think the .css can specify some things we should do by default, like re-size images to fit on page.

Reply:
These are for local instance theming - there is a file that applies globally though https://github.com/galaxyproject/galaxy/pull/8893/files#diff-b7164468c47f82e12c9ce4cc7ab669e6.

Reply:
Awesome, thanks!